jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/Query.java

```diff
@@ -116,11 +126,19 @@ public boolean isAllowDottedKeys() {
 		return allowDottedKeys;
 	}
 
+	public boolean isUseAllTypeNames() {
```

Comment:
It is true that I follow Java Bean conventions over English grammar, I tried to keep consistency. A few lines above there is a `isAllowDottedKeys` method, as opposed do `allowsDottedKeys`. Would you like me to

 - change all method names to correct English,
 - stick to the Java Bean convention,
 - make this method name an exception?

I will do as you prefer, but IMHO consistency is a value of good code.